lib/player.js

```diff
@@ -6570,6 +6535,37 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return config;
   }
 
+  /**
+   * @return {!Object}
+   * @private
+   */
+  defaultLowLatencyConfig_() {
```

Comment:
Should `getNonDefaultConfiguration()` print default LL values or not?

Reply:
How it works is that getNonDefaultConfiguration returns the configured values, so if it is a non-LL stream, it does not return any LL values, if the stream is LL, then it returns the current values ​​including the values ​​configured for LL.